apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/index.scss

```diff
@@ -0,0 +1,31 @@
+@import '~@automattic/page-template-modal/src/styles/page-template-modal';
+
+// Sidebar modal opener goo.
```

Comment:
goo?  :) As in glue that is holding something together?  That might not be clear to next dev who comes along. :)

Reply:
I see that was an existing comment....